targets/targets.json

```diff
@@ -7692,7 +7694,7 @@
     "FVP_MPS2": {
         "inherits": ["ARM_FM"],
         "public": false,
-        "supported_toolchains": ["GCC_ARM", "ARM", "IAR"],
+        "supported_toolchains": ["GCC_ARM", "ARMC5", "IAR"],
```

Comment:
FVP targets are failing with ARMC6 compiler, it appears to code itself is ok but the compiler fails with internal error/fault.
At this point, we dont know whats causing this and may need escalation to Compiler team and while we explore that we are using ARMC5 for FVP target.

Reply:
This needs tracking issue then

Reply:
@0xc0170 - Please see ARM Internal Ref: IOTCORE-1059